### PR TITLE
assimp 5.1.4

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by iMichka** [comment]
Might need a newer gcc on Linux

File: Formula/assimp.rb [hunk 1]
```diff
@@ -16,15 +16,27 @@ class Assimp < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "ninja" => :build
```
Comment (carlocab) [inline Formula/assimp.rb:19]: Do we need to use Ninja?

Comment (chenrui333) [reply Formula/assimp.rb:19]: Can you elaborate a bit?

Comment (carlocab) [reply Formula/assimp.rb:19]: Why not use `make` instead of `ninja`?

Comment (chenrui333) [reply Formula/assimp.rb:19]: let me give a try on that.

Comment (chenrui333) [reply Formula/assimp.rb:19]: actually did not make it work with `make`

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**